mmv1/third_party/terraform/website/docs/r/compute_security_policy.html.markdown

```diff
@@ -212,6 +279,12 @@ The following arguments are supported:
 
 * `interval_sec` - (Optional) Interval over which the threshold is computed.
 
+* <a  name="nested_exceed_redirect_options"></a>The `nested_exceed_redirect_options` block supports:
```

Comment:
I think the text here should read: "The `exceed_redirect_options` block supports". It looks like an extra `nested_` prefix was added.

Reply:
Good catch. I will change it.